automation/vars/uni02beta.yaml

```diff
@@ -1,9 +1,8 @@
 ---
 vas:
-  uni02beta:
+  uni02beta-adoption:
```

Comment:
shouldn't this be still uni02beta: ? This is uni02beta.yaml, the variables for uni02beta-adoption are in the file above, uni02beta-adoption.yaml.